src/simtools/applications/derive_photon_electron_spectrum.py

```diff
@@ -82,10 +82,24 @@ def _parse(label):
         required=False,
     )
     config.parser.add_argument(
-        "--threshold_afterpulse_spectrum",
-        help="Threshold for afterpulse ratio calculation",
+        "--afterpulse_amplitude_range",
+        help="Amplitude range in pe for afterpulse calculation",
```

Comment:
Is this range for calculating the afterpulse? (we don't actually calculate it.) 
Perhaps it is the range for fitting? In the other PR the limit from below was defaulted to 4.0 which makes more sense to me.

Reply:
- fit is done in the range of available data points (so no user input on this)
- `afterpulse_amplitude_range` is the range over which the fitted function is used to calculate afterpulses and forward it to norm_spe

Any suggestion on how to name it better? I don't have a good idea...

Reply:
Name is fine (or it could be `afterpulse_spectrum_range` if you want). Perhaps the point is about the word "calculation", because I don't expect we calculate the afterpulsing spectrum. Either way, it's not a big deal, you can leave it as is as well.